dozer/cogs/moderation.py

```diff
@@ -406,15 +407,16 @@ class Moderation(Cog):
             else:
                 user = Deafen(id=ctx.author.id, guild=ctx.guild.id, self_inflicted=True)
                 session.add(user)
+                await ctx.send("Deafening...")
```

Comment:
[Typing notifications](https://discordpy.readthedocs.io/en/rewrite/ext/commands/api.html#discord.ext.commands.Context.typing) would be a better way to show the command is in progress.